File: src/beginner/day18/Question02.java
package beginner.day18;

public class Question02 {

    public static class Solution {

        public int solution(int n) {

            return Math.sqrt(n) % 1 == 0 ? 1 : 2;
        }
    }

    public static void main(String[] args) {

        Solution solution = new Solution();

        System.out.println(solution.solution(144));
        System.out.println(solution.solution(976));
    }
}

File: src/beginner/day19/Question02.kt
package beginner.day19

class Question02{

    fun solution(my_str: String, n: Int): Array<String> {
        val strList = mutableListOf<String>()

        strList.addAll(my_str.chunked(n))

        return strList.toTypedArray()
    }
}

fun main() {

    val solution = Question02()

    println(solution.solution("abc1Addfggg4556b", 6).contentToString())
    println(solution.solution("abcdef123", 3).contentToString())
}